SimTracker/TrackerMaterialAnalysis/plugins/TrackingMaterialProducer.h

```diff
@@ -64,6 +64,11 @@ class TrackingMaterialProducer : public SimProducer,
   TFile* output_file_;
   TProfile* radLen_vs_eta_;
   bool isHGCal;
+  bool isHFNose;
+  static constexpr float innerHGCalEta = 2.4;
+  static constexpr float outherHGCalEta = 2.0;
+  static constexpr float innerHFnoseEta = 4.;
+  static constexpr float outherHFnoseEta = 3.3;
```

Comment:
If you have to make a new commit anyhow, I'd profit and fix the name of this parameter as ```outerHFnoseEta```